### PR TITLE
add hw-6+7

### PR DESCRIPTION
**Pull request by msemerni**

### CODE REVIEW

File: students/msemerni/hw-6+7/js/task-2.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//// Task 2
+//// Output in console all lowercase letters from English alphabet
+//// a
+//// b
+//// ...
+//// z
+
+function outputEnglishLetters1() {
+  let alphabet = 'abcdefghijklmnopqrstuvwxyz';
+  let alphabetArray = Array.from(alphabet)
```
Comment (Temu4) [inline students/msemerni/hw-6+7/js/task-2.js:10]: Why not `const`?

File: students/msemerni/hw-6+7/js/task-2.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  alphabetArray.forEach(letter =>
+    console.log(letter));
+}
+
+function outputEnglishLetters2() {
```
Comment (Temu4) [inline students/msemerni/hw-6+7/js/task-2.js:15]: Thank you! I've been waiting such approach 👍

File: students/msemerni/hw-6+7/js/task-3.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+//// Transform each element in the initial array with numbers
+//// that represent days of the week to an array with human readable day names
+
+//// first
+const numberDays = [3, 4, 5, 1, 6, 2, 7];
+
+function convertNumberDays1(daysArray) {
+  let humanDays = [];
+  daysArray.forEach(day => {
+    if (day === 1) {
```
Comment (Temu4) [inline students/msemerni/hw-6+7/js/task-3.js:11]: `switch` will definitely looks better here

File: students/msemerni/hw-6+7/js/task-3.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+  "Sunday",
+];
+
+function convertNumberDays2(daysArray, humanDaysArray) {
+  let humanDays = [];
+  daysArray.map((days) => {
```
Comment (Temu4) [inline students/msemerni/hw-6+7/js/task-3.js:44]: `Array.map` return new array. You shouldn’t create it somewhere before. [info](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: students/msemerni/hw-6+7/js/task-3.js (hunk 3 above)
Comment (Temu4) [reply students/msemerni/hw-6+7/js/task-3.js:11]: I've never heard that. Could you share a links?

**Review by Temu4 — APPROVED** [review]

File: students/msemerni/hw-6+7/js/task-3.js (hunk 3 above)
Comment (msemerni) [reply students/msemerni/hw-6+7/js/task-3.js:11]: https://www.google.com/search?q=don%27t+use+switch+statement+javascript&oq=dont+use+switch+&aqs=chrome.3.69i57j0i19l3j0i10i19i22i30l3j0i19i22i30j0i10i19i22i30.16514j0j4&client=ms-android-xiaomi-rev1&sourceid=chrome-mobile&ie=UTF-8#ip=1

Comment (Temu4) [reply students/msemerni/hw-6+7/js/task-3.js:11]: Yeap, but there are nothing about `never` using of `switch`. 
Definitely, there could be scenarios when it's better to replace `switch` to another constructions, but **in general it's absolutely OK to use it in production**.

**Pull request merged**